Dockerfile.build

```diff
@@ -0,0 +1,11 @@
+FROM gradle:5.4.1-jdk11
```

Comment:
We don't actually need a `gradle` image since we're running the build via the wrapper.